hubtools/src/caboose.rs

```diff
@@ -122,6 +128,22 @@ impl CabooseBuilder {
         self
     }
 
+    pub fn epoch<S: Into<String>>(mut self, epoch: S) -> Self {
```

Comment:
The field should be a `u32`, so this should be

```rust
pub fn epoch(mut self, epoch: u32) -> Self {
```

which makes a bunch of things cleaner

Reply:
Done